src/db-access.R

```diff
@@ -0,0 +1,18 @@
+#------------------ DATABASE TABLES-------------------#
+
+# db_usr is defined in .Renviron in the Home directory
+# db_pwd is defined in .Renviron in the Home directory
+
+
+conn <- RPostgreSQL::dbConnect(drv = RPostgreSQL::PostgreSQL(),
+                               dbname = "sdad",
+                               host = "postgis1",
+                               port = 5432,
+                               user = Sys.getenv(x = "db_usr"),
```

Comment:
I think we are going to use
```
conn <- dbConnect(drv = Postgres(),
                  dbname = "sdad",
                  user = Sys.getenv(x = "DB_USR"),
                  password = Sys.getenv(x = "DB_PWD"))
```
with the convention that constants are uppercase. Sorry!

Reply:
I fixed it!